examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,362 @@
+# correct.
+#
+# We can start by plotting the trajectory of the midpoint between the ears. We
+# will refer to this as the head trajectory.
+
+fig, ax = plt.subplots(1, 1)
+mouse_name = ds.individuals.values[0]
+
+sc = ax.scatter(
+    midpoint_ears.sel(individuals=mouse_name, space="x"),
+    midpoint_ears.sel(individuals=mouse_name, space="y"),
+    s=15,
+    c=midpoint_ears.time,
+    cmap="viridis",
+    marker="o",
+)
+
+ax.axis("equal")
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.invert_yaxis()
+ax.set_title(f"Head trajectory ({mouse_name})")
+fig.colorbar(sc, ax=ax, label=f"time ({ds.attrs['time_unit']})")
+fig.show()
+
+# %%
+# We can see that the majority of the head trajectory data is within a
+# cruciform shape. This is because the dataset is of a mouse moving on an
+# [Elevated Plus Maze](https://en.wikipedia.org/wiki/Elevated_plus_maze).
+
+# %%
+# Visualise the head vector
+# ---------------------------
+# To visually check our computation of the head vector, it is easier to select
+# a subset of the data. We can focus on the trajectory of the head when the
+# mouse is within a small rectangular area, and within a certain time window.
+
+# area of interest
+xmin, ymin = 600, 665  # pixels
+x_delta, y_delta = 125, 100  # pixels
+
+# time window
+time_window = range(1650, 1670)  # frames
+
+
+# %%
+# For that subset of the data, we now plot the head vector.
+
+fig, ax = plt.subplots(1, 1)
+mouse_name = ds.individuals.values[0]
+
+# plot midpoint between the ears, and color based on time
+sc = ax.scatter(
+    midpoint_ears.sel(individuals=mouse_name, space="x", time=time_window),
+    midpoint_ears.sel(individuals=mouse_name, space="y", time=time_window),
+    s=50,
+    c=midpoint_ears.time[time_window],
+    cmap="viridis",
+    marker="*",
+)
+
+# plot snout, and color based on time
+sc = ax.scatter(
+    position.sel(
+        individuals=mouse_name, space="x", time=time_window, keypoints="snout"
+    ),
+    position.sel(
+        individuals=mouse_name, space="y", time=time_window, keypoints="snout"
+    ),
+    s=50,
+    c=position.time[time_window],
+    cmap="viridis",
+    marker="o",
+)
+
+# plot the computed head vector
+ax.quiver(
+    midpoint_ears.sel(individuals=mouse_name, space="x", time=time_window),
+    midpoint_ears.sel(individuals=mouse_name, space="y", time=time_window),
+    head_vector.sel(individuals=mouse_name, space="x", time=time_window),
+    head_vector.sel(individuals=mouse_name, space="y", time=time_window),
+    angles="xy",
+    scale=1,
+    scale_units="xy",
+    headwidth=7,
+    headlength=9,
+    headaxislength=9,
+    color="gray",
+)
+
+ax.axis("equal")
+ax.set_xlim(xmin, xmin + x_delta)
+ax.set_ylim(ymin, ymin + y_delta)
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.set_title(f"Zoomed in to check ({mouse_name})")
+ax.invert_yaxis()
+fig.colorbar(sc, ax=ax, label=f"time ({ds.attrs['time_unit']})")
```

Comment:
the colorbar contains decimal-values ticks (.0 and .5) for some reason. That's confusing because the unit is frames, so it should be all integers, right?